settings.xml

```diff
@@ -0,0 +1,11 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Comment:
Why did you add this file to the root of the project and not the `.github` folder for example?